elyra/__init__.py

```diff
@@ -46,5 +46,7 @@ def load_jupyter_server_extension(nb_server_app):
         (url_path_join(web_app.settings['base_url'], r'/elyra/namespace'), NamespaceHandler),
         (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/schedule'), PipelineSchedulerHandler),
         (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/export'), PipelineExportHandler),
+        (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/config/%s' % (resource_regex)),
```

Comment:
I think I'd prefer an endpoint like `elyra/component` or `elyra/pipeline/component` rather than `config`.

I think we should return a dictionary of two items/keys:  'palette': palette.json, and 'properties': {dict of properties.json per node-type}.
```JSON
{
    "response": {
        "palette": {
            palette.json
        },
        "properties": {
            "notebookOp": {
                notebook-properties.json
            },
            "pythonScriptOp": {
                python-script-properties.json
            }
        }
    }
}
```